automatic/kitty/tools/chocolateyInstall.ps1

```diff
@@ -1,11 +1,35 @@
-﻿$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+StopProcesses
+
+$packageArgs = @{
+    PackageName    = 'kitty'
+    FileFullPath   = Get-Item $toolsPath\kitty-bin-*.zip
+    Destination    = $toolsPath
+  }
+
+Get-ChocolateyUnzip @packageArgs
+Remove-Item -force "$toolsPath\*.zip" -ea 0
+Remove-Item -force "$toolsPath\kitty-beta.exe" -ea 0
+
+# Remove kitty.exe to replace with either kitty_nocompress.exe or kitty_portable.exe
+Remove-Item -force "$toolsPath\kitty.exe" -ea 0
```

Comment:
These calls to `Remove-Item` can be combined into one call.

```suggestion
Remove-Item -force "$toolsPath\*.zip", "$toolsPath\kitty-beta.exe", "$toolsPath\kitty.exe" -ea 0
```

Reply:
Your point about the comment is fair, and I have updated my suggestion to keep it included.

However, having multiple remove-item calls when it is not needed is unnecessary, no matter how you look at it. So that suggestion is still relevant.

Reply:
> when it is not needed is unnecessary

Its necessary, however you frame it, because semantics matter. By that logic, we can keep entire script in the same line because multiline code is not necessary, no matter how you look at it.